### ISSUE TITLE
DisplayTint: support external monitors/brightness in OS 14

### ISSUE THREAD

**Issue by taramk** [darwinkitdev/upwork-demos#15]
Tested with a laptop connected to one external monitor.

On OS 13/Intel chip:
* brightness is only applied to laptop screen
* tint is only applied to main display (whether it's the laptop screen or an external display)

On OS 14/Apple Silicon:
* brightness is not applied at all
* tint is only applied to main display (whether it's the laptop screen or an external display)

**Comment by ericmaciel** [comment]
@taramk

The tint was fixed for multiple screens.

I managed to fix the brightness for Apple Silicon just by switching API. Before, I was using a public but deprecated API. Now I'm using a private API that works on Apple Silicon too. This solution should also work for external Apple displays.

Brightness still doesn't work on external displays from other manufacturers. In order for it to work, I'll need to implement the DDC/CI protocol to control external displays. The implementation doesn't seem difficult, but it can add a fair amount of code. I'm still looking into it.
